modules/addtask.js

```diff
@@ -0,0 +1,23 @@
+const todoList = document.getElementById('todoList');
+
+module.exports = function addtask(boo) {
+  todoList.innerHTML += `
+    <div class="todoFlex">
+      <div class="todoDiv">
+        <input type="checkbox" name="${boo.id}" class="checkbox" ${boo.completed ? 'checked' : ''} >
+        <input class="todoP" name="${boo.id}" value="${boo.description}" >
+      </div>
+      <i class="bi bi-three-dots-vertical dots"></i>
+    </div>
+  `;
+
+  if (localStorage.getItem('tasks') === null) {
+    const tasks = [];
+    tasks.push(boo);
+    localStorage.setItem('tasks', JSON.stringify(tasks));
```

Comment:
[OPTIONAL]

I will strongly recommend you create a separate function that stores and retrieve data from the local storage to avoid repeating yourself. You can always call the function when needed 😄